src/Post.php

```diff
@@ -47,7 +47,6 @@
  *     </div>
  * </article>
  * ```
- * @todo implement JsonSerializable?
```

Comment:
Serialization can be added if needed. It’s documented in https://timber.github.io/docs/v2/guides/posts/#serialization.